lib/display.ts

```diff
@@ -0,0 +1,175 @@
+import { createFromJSON, DepGraph } from "@snyk/dep-graph";
+import chalk from "chalk";
+
+import { Issue, Options, ScanResult, TestResult } from "./types";
+
+export async function display(
+  scanResults: ScanResult[],
+  testResults: TestResult[],
+  errors: string[],
+  options?: Options,
+): Promise<string> {
+  const testingPath = options?.path;
+
+  const result: string[] = [];
+  if (options && options.path) {
+    const prefix = chalk.bold.white("\nTesting " + testingPath + "...\n\n\n");
+    result.push(prefix);
+  }
+
+  for (const testResult of testResults) {
+    const formattedIssue: string[] = [];
+    for (const issue of testResult.issues) {
+      formattedIssue.push(formatIssue(testResult, issue));
+    }
+    result.push(formattedIssue.join("\n"));
+    result.push("");
+
+    const metadata = formatMetadataSection(testResult);
+    result.push(metadata);
+
+    const summary = formatSummary(testResult);
+    result.push(summary);
+  }
+
+  return result.join("\n");
+}
+
+function formatIssue(testResult: TestResult, issue: Issue): string {
+  const result: string[] = [];
+  const issueData = testResult.issuesData[issue.issueId];
+  const severity = capitalize(issueData.severity);
+  const pkg = issue.pkgName;
+  const color = getColor(issueData.severity);
+
+  const header = color(`✗ ${severity} severity vulnerability found in ${pkg}`);
+  const description = `  Description: ${issueData.title}`;
+  const info = `  Info: https://snyk.io/vuln/${issue.issueId}`;
```

Comment:
I wonder if the URL should be read from the one stored in `snyk config`, maybe not something to cover for now.

Reply:
not sure we need to - what was the cli doing so far? we should support exactly what it used to